src/client/components/footerChatProfile/FooterChatProfile.js

```diff
@@ -1,19 +1,39 @@
 import React from 'react';
 import './FooterChatProfile.styles.css';
+import { NavLink } from 'react-router-dom';
 
-export default function FooterChatProfile() {
-  return (
-    <div className="footer-chat-profile">
-      <div className="title-dot">
-        <a className="overview-active-link" href="/Overview">
-          Chats
-        </a>
-        <span className="chat-dot"> </span>
+const FooterChatProfile = () => (
```

Comment:
Does this not need to take in some kind of prop indicating which nav is the active one? Like is it on profile or the overview page. Then we add the small circle in the bottom

Reply:
no, I removed it. In React it works with activeClassName. I based on this article https://medium.com/@rdomanoel/how-to-create-an-active-classname-navbar-using-react-and-react-router-70d890c9b58e

Reply:
But yes, before I was thinking in your way :)